apps/schools/models/partners.py

```diff
@@ -93,8 +93,8 @@ class DiseInfo(BaseModel):
     http://www.klp.org.in/schoolpage/school/33166?tab=finances
     '''
     dise_code = models.CharField(max_length=32, primary_key=True)
-    assembly = models.ForeignKey('Assembly')
-    parliament = models.ForeignKey('Parliament')
+    # assembly = models.ForeignKey('Assembly')
+    # parliament = models.ForeignKey('Parliament')
```

Comment:
The `assembly` and `parliament` data are a bit of a mystery at this point because on `dubdubdub`, it is an mview populated from the `spatial` db. The origins of where this data on the `spatial` db came from is under investigation. Once that is figured out, we should probably populate that for Odisha and then get these two fields in.